samples/cloudwatch/README.md

```diff
@@ -2,52 +2,75 @@
+Follow these steps to deploy CloudWatch and enable it to collect metrics from
+SAS Viya:
 
 1. Attach CloudWatchAgentServerPolicy to IAM roles
 2. Identify the cluster and region
 3. Deploy the CloudWatch Prometheus agent
 4. Apply SAS Viya customizations
-  * Customize the Prometheus scrape configuration
-  * Customize the mapping of performance log events to CloudWatch metrics
+    - Customize the Prometheus scrape configuration
+    - Customize the mapping of performance log events to CloudWatch metrics
 
 ### Attach CloudWatchAgentServerPolicy to IAM Roles
 
 The CloudWatch agent requires proper permissions to create performance log
-events and CloudWatch metrics. AWS uses Identity and Access Management (IAM) roles to set these permissions. See the [IAM Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
+events and CloudWatch metrics. AWS uses Identity and Access Management (IAM)
+roles to set these permissions. See the
+[IAM Roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html)
 documentation for more information.
 
 Follow these steps to set permissions:
 
 1. Sign in to the AWS Management Console and open the IAM console.
 2. In the navigation pane of the console, select `Roles`.
-3. In the Search field, enter the name of the Amazon EKS cluster on which SAS Viya is deployed (such as `my-cluster`) to filter the roles listed.
-4. Select the role that contains `AWS service: eks` in the `Trusted entities` column.
+3. In the Search field, enter the name of the Amazon EKS cluster on which SAS
+Viya is deployed (such as `my-cluster`) to filter the roles listed.
+4. Select the role that contains `AWS service: eks` in the `Trusted entities`
+column.
 5. Select `Attach Policies`.
-6. Search for `CloudWatchAgentServerPolicy`. If the `CloudWatchAgentServerPolicy` does not exist, see:
+6. Search for `CloudWatchAgentServerPolicy`. If the
+`CloudWatchAgentServerPolicy` does not exist, see:
 [Creating IAM Roles for Use with the CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-iam-roles-for-cloudwatch-agent-commandline.html).
 7. Select the check box next to `CloudWatchAgentServerPolicy`.
 8. Select `Attach Policy`.
-9. Repeat steps 4 through 8 for the role that contains `AWS service: ec2` in the `Trusted entities` column.  
+9. Repeat steps 4 through 8 for the role that contains `AWS service: ec2` in
+the `Trusted entities` column.  
 
 ### Identify Cluster and Region
 
-You must identify the Amazon EKS cluster on which SAS Viya is deployed and the region for the cluster. These values are needed to construct the correct endpoint for CloudWatch.
+You must identify the Amazon EKS cluster on which SAS Viya is deployed and the
+region for the cluster. These values are needed to construct the correct
+endpoint for CloudWatch.
 
 Run the following code from the command line. Replace the
 values of `ClusterName` and `RegionName` with the values for your EKS cluster.
```

Comment:
Rather than saying "run the following code", maybe say _"Set the ClusterName and RegionName environment variables with the values for your EKS cluster"_  While setting env vars may, technically, be "submitting code", that seems to overstate things.
And are we only setting env vars so users can cut-and-paste the curl command shown afterwards?  Or, are the env vars needed/used later?  If it is the former, I think we should point that out (e.g. "having set the environment variables appropriately for your cluster, you can cut-and-paste the following command to...").

Reply:
Wording changed as suggested